lib/datadog/tracing/contrib/rails/runner.rb

```diff
@@ -105,7 +105,7 @@ def self.load(file, wrap = true)
                 end
                 Contrib::Analytics.set_rate!(span, Datadog.configuration.tracing[:rails])
 
-                super
+                Kernel.load(file, wrap)
```

Comment:
I would not change `super` to `Kernel.load` - there might be other modules prepended to `Kernel`, and we probably wouldn't want to skip them